tgui/.prettierrc.yml

```diff
@@ -1,8 +1,9 @@
 arrowParens: always
-bracketSpacing: true
```

Comment:
Why did you remove bracket spacing?

Reply:
This does not exist on prettierx apparently, or at least is categorized into subtypes

Reply:
i'd still keep it

Reply:
It throws a warning on ci, can we  use one of its granular subtypes?